tgui/packages/tgui/styles/interfaces/Accounting.scss

```diff
@@ -0,0 +1,30 @@
+//Originally Authored by IAmGoofball
```

Comment:
actually we stole it from some random html website talking about why blink was disabled

Reply:
@ArcaneMusic please move contents of this file to `tgui/tgui.html` (since it's a very generic polyfill), put it at the beginning of the `<style>` block. Below is a more compact form that you should use.

```css
blink {
  animation: 1s linear infinite condemned-blink-effect;
}

@keyframes condemned-blink-effect {
  0% { visibility: hidden; }
  50% { visibility: hidden; }
  100% { visibility: visible; }
}
```

In `tgui/global.d.ts`, after modules and before TguiMessage and ByondType, insert this:

```tsx
namespace JSX {
  interface IntrinsicElements {
    marquee: any;
    blink: any;
  }
}
```